score-analysis/resources/tools/DB12.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```

Comment:
> You have added the results of the score analysis in, which are of course rather large files, that I don't think deserves being kept here (they are not for being version-controlled). If you want to save them, maybe somewhere else?

Yes I agree, `Mendeley Data` or `Zenodo` might be a better place for the `score-analysis` directory. Should I move the directory to one of these platforms and just add a simple link to it in this repo? Any other option that you consider for these files?

> do you really need to have this one? Isn't it an almost copy-past of what is in https://github.com/DIRACGrid/DB12/blob/master/src/db12/benchmark.py ?

Indeed, I wanted to remove it, this function is very similar to `single_dirac_benchmark()` but do not correct the norm.
I could add a flag to the CLI to prevent correcting the norm in `single_dirac_benchmark()` and thus we would not need this function anymore. What do you think?

Reply:
I agree on both your suggestions.